docs/scope.md

```diff
@@ -0,0 +1,113 @@
+    - [Discovery](#discovery)
+  - [Clusters](#clusters)
+  - [Environments](#environments)
+  - [Landscapes](#landscapes)
+  - [SAP Systems](#sap-systems)
+  - [Agents](#agents)
+
+# About this document
+This document aims to provide information about the current features that `trento`
+tries to cover and it's main scope. Further, we provide details on the functionality
+provided by each view.
+
+## Our scope
+>Provide simple to use front end for all SAP relevant OS related tasks for SAP 
+>workloads
+
+This means that `trento` focuses on the OS-related tasks without stepping into 
+specific functionality that is already covered by the relevant application layer,
+centered around SAP workloads and simplicity.
+
+## Hosts
+In the hosts overview, the user can get a list of all the hosts that are running
+the trento agent. For each host it will be possible to check basic information
+such as the hostname, it's IP address, to what cluster it belongs and a list of
+all the tags that have been set by the discovery mechanisms to classify each
+host accordingly. 
+
+When accessing the details of each host, it will be possible to access in-depth
+information about the role each host has in the cluster such as the status of the
+core HDB processes, SAP System ID (SID), etc. Here, the SAP administrator can
+get a list of potential problems and improvements on the configuration of the
+host that the discovery mechanisms have found.
+
+### Checks
+> TODO: Put some additional details here.
+
+We currently support 3 checkers:
+
+> FIXME: I don't know how to describe more in detail how the checks UI are 
+> presented, e.g. why some are clickable.
+> It also feels to me that we are mixing `checkers` and `discovery` in the UI,
+> shouldn't they be separated?
+
+#### HA Checker: Scenario SAP HANA Performance Optimized on Azure
+> FIXME: How do I releate this "HA Checker: Scenario SAP HANA Performance 
+> Optimized on Azure"
+This checker uses Azure's recommendations to provide an overview of how well
+the deployed SAP HANA cluster meets their indicators. More details about this
+recommendation can be seen [here](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker)
+
+#### HA Checker: Scenario Pacemaker on Azure
+This check is also comparing against Azure's recommendations for a Pacemaker
+cluster.
+> FIXME: Link to this one?
+
+#### More coming
+We are working on implementing more checkers also for other cloud providers.
+
+### Discovery
+> FIXME: Based on what I wrote in [Checks](#checks) I think we should probably
+> have a separate part here: Discovery are the parts of trento which are
+> attempting to detect e.g. the role of a SAP HANA node and the processes that
+> run on each node; while the checkers are additionally checking those parameters
+> against predefined values.
+
+## Clusters
+This view currently allows to check the status of the all the discovered HANA DB
```

Comment:
I would remove `HANA DB` here and maybe only mention clusters. We will have SAP app clusters eventually (and maybe more on the road)